criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
+| Build & Release | All software releases MUST be assigned a standardized unique version identifier. |
+| Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS and TLS. |
+| Documentation | The project MUST have one or more mechanisms for public discussions about proposed changes and usage obstacles. |
+| Documentation | The project documentation MUST provide user guides for all basic functionality. |
+| Documentation | The project documentation MUST include an explanation of the contribution process. |
+| Quality | The version control system MUST be publicly readable and have a static URL. |
```

Comment:
What do you mean by static URL?   What is a common counterexample?

Reply:
We'll need to elaborate in the justification/details content— but the idea presented here is to ensure that the project is using standard reasonable mechanisms for the VCS.

Counter example would be a project changing it's URL— it happens, but it should raise a flag and definitely shouldn't be routine. Or if a project mirrors between two locations and doesn't have a consistent stable location where development is happening.

(TBH I have no attachment to "static," and I don't recall where that suggestion originally came from)